ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -137,8 +137,11 @@ open class PillButtonBar: UIScrollView {
         }
     }
 
-    @objc public init(pillButtonStyle: PillButtonStyle = .outline) {
+    private var customPillButtonBackgroundColor:UIColor?
```

Comment:
Have you ran swiftlint on the files? There should be a space between the colon and "UIColor":

private var customPillButtonBackgroundColor: UIColor?

Reply:
Also, can you move this close to the other private vars (above)?

Reply:
Example of swiftlint usage on a file:

swiftlint autocorrect --path /Volumes/Source/git/fluentui-apple/ios/FluentUI/Pill\ Button\ Bar/PillButtonBar.swift

Reply:
Ran swiftlint on files and updated.

---
In reply to: [468707275](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468707275) [](ancestors = 468707275)

Reply:
ran swiftlint and updated

---
In reply to: [468710216](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468710216) [](ancestors = 468710216)

Reply:
moved

---
In reply to: [468708189](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468708189) [](ancestors = 468708189)